distributed/rpc/pipeline/main.py

```diff
@@ -248,6 +207,9 @@ def run_master(split_size):
         labels = torch.zeros(batch_size, num_classes) \
                       .scatter_(1, one_hot_indices, 1)
 
+        # The distributed autograd context is the deciated scope for the
```

Comment:
nit: was this supposed to be "dedicated scope"?

Reply:
good catch! Thanks!